Ada_Zangemann-cs.txt

```diff
@@ -5,181 +5,123 @@
 
-The information in this book is distributed on an “As Is” basis, without 
-warranty. While every precaution has been taken in the preparation of 
-this work, neither the author nor No Starch Press, Inc. shall have any 
-liability to any person or entity with respect to any loss or damage 
-caused or alleged to be caused directly or indirectly by the information 
-contained in it.
+Informace v této knize jsou poskytovány „tak, jak jsou“, bez záruky. Ačkoli při přípravě této práce byla přijata veškerá opatření, autor ani No Starch Press, Inc. nenesou žádnou odpovědnost vůči jakékoli osobě či subjektu za jakoukoli ztrátu nebo škodu způsobenou či údajně způsobenou přímo nebo nepřímo informacemi v ní obsaženými.
 
-All characters in this publication are fictitious, and any resemblance 
-to real persons, living or dead, is purely coincidental.
+Všechny postavy v této publikaci jsou smyšlené a jakákoli podoba s reálnými osobami, živými či mrtvými, je čistě náhodná.
 
 **
 
-This book by Matthias Kirschner (author) and Sandra Brandstätter 
-(illustrator) is licensed under <b>Creative Commons CC-BY-SA</b>, a free 
-cultural license.
-
-<b>You may</b> share, copy, and redistribute the material in any medium 
-or format. <b>You may</b> adapt, remix, transform, and build upon the 
-material for any purpose, even commercially. <b>You must</b> give 
-appropriate credit, provide a link to the license, and indicate if 
-changes were made.
-<b>You may</b> do so in any reasonable manner, but not in any way that 
-suggests the licensor endorses you or your use. If you remix, transform, 
-or build upon the material, <b>you must</b> distribute your 
-contributions under the same license as the original. (This is a 
-human-readable summary of - and not a substitute for - the “legal code” 
-of CC-BY-SA 4.0.)
-
-Furthermore for remix versions (derivative works), the licensor asks you 
-not to make any modifications of the author's personal acknowledgments 
-and the author's and illustrator's short biographies, instead please 
-completely remove those parts from your remix version.
-
-Publication under this free cultural license was possible due to the 
-great support by dpunkt.verlag GmbH / O'Reilly Media, Inc. for the 
-German book and No Starch Press for the English book. Unfortunately, 
-Creative Commons licenses are still rare exceptions for commercially 
-published books, and often considered revolutionary and risky by 
-publishers.
-
-<b>A big thank you</b> to <b>dpunkt.verlag</b> and </small><small><b>No 
-Starch Press</b></small><small> for understanding the values of software 
-freedom, taking the time to discuss and understand the advantages of 
-free cultural licenses, and the great support to publish this book under 
-conditions that respect the FSFE's values.
+Tato kniha od Matthiase Kirschnera (autor) a Sandry Brandstätterové (ilustrátorka) je licencována pod licencí <b>Creative Commons CC-BY-SA</b>, což je svobodná kulturní licence.
```

Comment:
```suggestion
Tato kniha od Matthiase Kirschnera (autor) a Sandry Brandstätterové (ilustrátorka) je licencována pod <b>Creative Commons CC-BY-SA</b>, což je svobodná kulturní licence.
```

Reply:
Dal bych tam raději "chráněna licencí"

Reply:
Jo, no to je pravda. Ovšem to sloveso je zase takové nepěkné. Tak "licenci" vyhodit a zbude jenom "chráněna Common ..."